mappings/net/minecraft/entity/passive/FoxEntity.mapping

```diff
@@ -123,16 +123,16 @@ CLASS net/minecraft/unmapped/C_axinhusn net/minecraft/entity/passive/FoxEntity
 		FIELD f_vvpvjvie type Lnet/minecraft/unmapped/C_axinhusn$C_qecgztkz;
 		METHOD <init> (Lnet/minecraft/unmapped/C_axinhusn$C_qecgztkz;)V
 			ARG 1 type
-	CLASS C_jdyjcdzs PickBerriesGoal
-		FIELD f_cfkermxf PICKING_TIME I
+	CLASS C_jdyjcdzs EatSweetBerriesGoal
+		FIELD f_cfkermxf EATING_TIME I
 		FIELD f_sojxnnoc timer I
 		METHOD <init> (Lnet/minecraft/unmapped/C_axinhusn;DII)V
 			ARG 2 speed
 			ARG 4 range
 			ARG 5 maxYDifference
 		METHOD m_kqvohfdd pickSweetBerries (Lnet/minecraft/unmapped/C_txtbiemp;)V
 			ARG 1 state
-		METHOD m_olxxahnm pickFromTargetBlock ()V
+		METHOD m_olxxahnm eatSweetBerry ()V
```

Comment:
Is this specific to the berry block?

Reply:
> 
> 
> Is this specific to the berry block?

They are specific to berries, but not exclusive to sweet berry bushes. Glow berries exist and this goal also picks them from glow vines

Also, I don't understand why the entirely of #15 was silently reverted here; Was it because the original names were on Mojmap? I think there is room for improvement (In the hindsight, `pickFromTargetBlock` should be something else, like `pickFromTargetPos`, or perhaps something simpler like `attemptBerryPick`?), but as I've mentioned on that PR, the original names don't work well

Reply:
I agree with you completely so maybe it should just be eatBerry then?

Reply:
Just anything other than eatSweetBerry

Reply:
I was going to say that it should be `eatFavoriteFood` since it's also items used for breeding, but turns out that fox's breeding item is actually data-driven with a tag, so that assumption is kinda false then.

So yeah, agreeing with @Bubblie01, `sweet` should be dropped for just berry.

Reply:
Pinging me on GitHub smh LOL but yeah changing it to eatBerry makes sense as this is specific to berries not just sweet berry we could also do eatBerries but that might be too confusing

Reply:
I still think that `Pick` is a more fitting name than `Eat`; When taking account of similar goals that does have "Eat" in the name and do directly involve, well, consuming food (ex: rabbits with EatCarrotCropGoal and sheep with EatGrassGoal), foxes would be the odd one since unlike others, it doesn't directly involve eating, and I think that would be confusing

This is only my opinion though; If we do stick with `Eat`, then yeah, the "sweet" part should at least be dropped

Reply:
Eat is indeed weird since the fox effectively picks berries, then the goal stop, then another goal for actually eating starts later.

Reply:
ok pickBerry would be better than in this case I didn't know there was a separate goal for eating

Reply:
My bad, I ported some names from yarn and forgot to revert these changes

Reply:
Fixed